pkg/internal/checkup/checkup.go

```diff
@@ -201,6 +213,19 @@ func (c *Checkup) Results() status.Results {
 	return c.results
 }
 
+func (c *Checkup) createVMIUnderTestCM(ctx context.Context) error {
+	log.Printf("Creating ConfigMap %q...", ObjectFullName(c.namespace, c.vmiUnderTestConfigMap.Name))
+
+	_, err := c.client.CreateConfigMap(ctx, c.namespace, c.vmiUnderTestConfigMap)
+	return err
+}
+
+func (c *Checkup) deleteVMIUnderTestCM(ctx context.Context) error {
+	log.Printf("Deleting ConfigMap %q...", ObjectFullName(c.namespace, c.vmiUnderTestConfigMap.Name))
+
+	return c.client.DeleteConfigMap(ctx, c.namespace, c.vmiUnderTestConfigMap.Name)
+}
```

Comment:
Please consider adjusting the signatures of `createTrafficGenCM` and `deleteTrafficGenCM` so they could be shared between the two configMaps.

In a similar way `createVMI` and `deleteVMI` work.